client/src/components/Cell/CellMain.tsx

```diff
@@ -36,12 +40,32 @@ const CellMain = ({
   const Confirm = useConfirm();
   const Alert = useAlert();
   const { brute, owner } = useBrute();
-
+  const { user } = useAuth();
+  const navigate = useNavigate();
   const xpNeededForNextLevel = useMemo(
     () => (brute ? getXPNeeded(brute.level + 1) : 0),
     [brute],
   );
-
+  const arrowProps = {
+    maxHeight: '70px', maxWidth: '70px', color: '#7d4736', cursor: 'pointer', fontSize: '150px', margin: 'auto'
+  };
+  const bruteList : string[] | undefined = user?.brutes.map((bruteDetails) => bruteDetails.name);
+  const swipeBrute = (direction : string) => {
```

Comment:
This logic needs to change.

On a left or right press, you first need to find the index of the current brute in the user brutes list.
On a left press, you need to retrieve the name of the previous brute (modulo brute count, so that if you press left on the first brute, you go to the last one), for ex `user.brutes[(currentBruteIndex - 1) % user.brutes.length]`
Same thing on the right press.

Also, never use `!` in Typescript, this is a banned notation, check for the existence of the prop higher up instead.